README.md

```diff
@@ -0,0 +1,29 @@
+# OpenSAFELY Command Line Interface
+
+_User-facing documentation to go here_
+
+
+## Developer docs
+
+To minimise the possibility for installation issues this package vendors
+all its dependencies under the [opensafely._vendor](./opensafely/_vendor)
+namespace using the [vendoring](https://pypi.org/project/vendoring/) tool.
+
+Note that while the `opensafely-jobrunner` package technically depends
+on `requests` this is only used by the `jobrunner.sync` module which is
+never invoked by this interface (as smoketests will confirm).
+
+To update the vendored dependencies first update the versions in
+[vendor.txt](./vendor.txt).
```

Comment:
What is the story for deciding what to vendor? If a new dependency is added to jobrunner which needs vendoring, I guess we're relying on the smoke tests to tell us that it's missing? If so, worth saying so here? Perhaps also saying "run tests locally aftering running vendoring tool"?

Reply:
We're now vendoring all the dependencies so I think this question goes away.